ansible/Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM alpine:3.8
+FROM python:2.7-alpine3.8
 
-ARG CI_HELPER_VERSION
-ARG PYTHON_VERSION
 ARG VERSION
+ARG CI_HELPER_VERSION
 
 WORKDIR /root
 RUN echo "Install Ansible with deps" && \
-    apk --update --no-cache add bash ca-certificates curl git openssh-client python=${PYTHON_VERSION} py-pip ansible=${VERSION} && \
-    echo "Done install Ansible with deps" &&\
+    apk add --update -q --no-cache bash build-base curl git libffi-dev make openssh-client rsync tzdata openssl-dev && \
+    pip install ansible==${VERSION} && \
+    echo "Done install Ansible with deps" && \
     echo "Install CI Helper" && \
-    curl -sSL https://github.com/rande/gitlab-ci-helper/releases/download/${CI_HELPER_VERSION}/alpine-amd64-gitlab-ci-helper -o /usr/bin/ci-helper && \
+    curl -sSL https://github.com/rande/gitlab-ci-helper/releases/download/${CI_HELPER_VERSION}/linux-amd64-gitlab-ci-helper -o /usr/bin/ci-helper && \
```

Comment:
Why change the distribution? It's still an Alpine-based image.

Reply:
Sorry, copy/paste failure :)